modules/web/test/com/company/touraddondemo/TourUiTest.groovy

```diff
@@ -73,6 +76,30 @@ class TourUiTest {
         close()
     }
 
+    @Test
+    void goThroughEditorTourTwice() {
+        clearDb()
```

Comment:
Move the methods for which it is necessary to clear a DB in a separate class. Use the @BeforeTest annotation for the `clearDb()` method.